### PR TITLE
add review section

### PR DESCRIPTION
**Pull request by MohitBansal321**
## Description

Please include a summary of the change and which issue is fixed. Also include relevant motivation & further context of what the PR is all about. List any dependencies that are required for this change.

Fixes #(Insert Issue Here Without Brackets)
#7 
## Changes Include

- [x] New feature ✨ (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
  I run it locally and check on all points which are best for it
  screenshot provide you what changes i do here 
- [ ] Test B

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

## Screenshots (if any)
Screenshot 2023-02-07 121337

(prefer animated gif)

### CODE REVIEW

File: templates/resources.html [hunk 1]
```diff
@@ -51,6 +51,15 @@
                 </div>
                 <span class="tag is-danger">{{res.level}}</span>
                 <span class="tag is-info">{{res.type}}</span>
+                <div class="content">
+                    {% if res.reviews %}
+                        <hr />
+                        <h5 class="subtitle is-5">Reviews</h5>
+                        {% for review in res.reviews %}
+                        <p>"{{review}}"</p>
```
Comment (Bhupesh-V) [inline templates/resources.html:59]: Do you think italicizing it will make the review actually look like a review? Like how we see reviews on products?

**Review by Bhupesh-V — CHANGES_REQUESTED** [review]
Looks good, just make sure to indent the code (using a jinja compatible formatter)

File: templates/resources.html (hunk 1 above)
Comment (MohitBansal321) [reply templates/resources.html:59]: I think it is okay for it and you mention me it is above the open link it if any  changes need let me know

**Comment by MohitBansal321** [comment]
> Looks good, just make sure to indent the code (using a jinja compatible formatter)

okay

**Comment by Bhupesh-V** [comment]
> > Looks good, just make sure to indent the code (using a jinja compatible formatter)
> 
> can I do with help of prettier?

I am not sure whether prettier will take in account of jinja templates, try using this maybe? https://github.com/wholroyd/vscode-jinja

**Review by Bhupesh-V — APPROVED** [review]

**Comment by Bhupesh-V** [comment]
@all-contributors please add @MohitBansal321  for code

**Pull request merged**